region/maridia/outer/Mt. Everest.json

```diff
@@ -3097,10 +3621,48 @@
         "Opening the door first and moonfalling can help but is not required."
       ],
       "devNote": [
-        "It's possible to get 40 frames remaining by opening the door first and dropping with a moonfall, but is it worth putting here?",
-        "A similar strat could be added from 7->3."
+        "It's possible to get 40 frames remaining by opening the door first and dropping with a moonfall, but is it worth putting here?"
       ]
     },
+    {
+      "link": [10, 3],
+      "name": "Leave With Temporary Blue",
+      "requires": [
+        "Gravity",
+        {"canShineCharge": {
+          "usedTiles": 20,
+          "gentleUpTiles": 3,
+          "gentleDownTiles": 1,
+          "steepUpTiles": 1,
+          "openEnd": 0
+        }}
+      ],
+      "exitCondition": {
+        "leaveWithTemporaryBlue": {
+          "direction": "left"
+        }
+      },
+      "unlocksDoors": [
+        {"types": ["super"], "requires": []},
+        {"types": ["missiles", "powerbomb"], "requires": [
```

Comment:
You can land on the door and open it while chainingTempBlue.  It could help compared to falling through at full speed too.